go.mod

```diff
@@ -3,25 +3,54 @@ module github.com/giantswarm/chart-operator
 go 1.13
 
 require (
-	github.com/giantswarm/apiextensions v0.1.1
+	github.com/giantswarm/apiextensions v0.2.0
 	github.com/giantswarm/appcatalog v0.1.11
-	github.com/giantswarm/backoff v0.0.0-20200209120535-b7cb1852522d
-	github.com/giantswarm/e2esetup v0.0.0-20191209131007-01b9f9061692
-	github.com/giantswarm/exporterkit v0.0.0-20190619131829-9749deade60f
-	github.com/giantswarm/helmclient v0.0.0-20200317180111-3fa03f5d7b76
-	github.com/giantswarm/k8sclient v0.0.0-20200120104955-1542917096d6
-	github.com/giantswarm/microendpoint v0.0.0-20200205204116-c2c5b3af4bdb
+	github.com/giantswarm/backoff v0.2.0
+	github.com/giantswarm/e2esetup v0.1.0
+	github.com/giantswarm/exporterkit v0.2.0
+	github.com/giantswarm/helmclient v0.0.0-20200331110847-d697c5c4a0fb
+	github.com/giantswarm/k8sclient v0.2.0
+	github.com/giantswarm/microendpoint v0.2.0
 	github.com/giantswarm/microerror v0.2.0
-	github.com/giantswarm/microkit v0.0.0-20191023091504-429e22e73d3e
-	github.com/giantswarm/micrologger v0.2.0
-	github.com/giantswarm/operatorkit v0.0.0-20200205163802-6b6e6b2c208b
-	github.com/giantswarm/versionbundle v0.0.0-20200205145509-6772c2bc7b34
+	github.com/giantswarm/microkit v0.2.0
+	github.com/giantswarm/micrologger v0.3.1
+	github.com/giantswarm/operatorkit v0.2.0
+	github.com/giantswarm/versionbundle v0.2.0
 	github.com/google/go-cmp v0.4.0
-	github.com/prometheus/client_golang v1.0.0
+	github.com/prometheus/client_golang v1.3.0
 	github.com/spf13/afero v1.2.2
 	github.com/spf13/viper v1.6.2
-	k8s.io/api v0.17.3
-	k8s.io/apimachinery v0.17.3
-	k8s.io/client-go v0.17.3
+	k8s.io/api v0.17.2
+	k8s.io/apimachinery v0.17.2
+	k8s.io/client-go v0.17.2
 	sigs.k8s.io/yaml v1.1.0
 )
+
+// Workaround for https://github.com/sirupsen/logrus/issues/570
+// Additional reading: https://github.com/golang/go/issues/28489
+// Solution: https://github.com/moby/moby/issues/39302
+replace github.com/docker/docker => github.com/docker/engine v1.4.2-0.20191113042239-ea84732a7725
+
```

Comment:
We shouldn't need all this replace stuff.

Its only needed for Helm 2 because it imports `k8s.io/kubernetes`. This branch is already using helm 3.